florist/api/db/entities.py

```diff
@@ -62,7 +62,7 @@ class Config:
                 "redis_host": "localhost",
                 "redis_port": "6380",
                 "uuid": "0c316680-1375-4e07-84c3-a732a2e6d03f",
-                "metrics": '{"type": "client", "initialized": "2024-03-25 11:20:56.819569", "rounds": {"1": {"fit_start": "2024-03-25 11:20:56.827081"}}}',
+                "metrics": '{"host_type": "client", "initialized": "2024-03-25 11:20:56.819569", "rounds": {"1": {"fit_start": "2024-03-25 11:20:56.827081"}}}',
```

Comment:
Did we change the key in the reporter here? If so it needs to be changed in the UI code as well.

Reply:
More specifically, here:
https://github.com/VectorInstitute/FLorist/blob/main/florist/app/jobs/details/page.tsx#L194-L199

and here:
https://github.com/VectorInstitute/FLorist/blob/main/florist/app/jobs/details/page.tsx#L290-L297

Reply:
@lotif Here as well https://github.com/VectorInstitute/FLorist/blob/d0604c4da538c09b3a117ae428c064a94075e19a/florist/app/jobs/details/page.tsx#L452? or is this unrelated

Reply:
Yes, I believe there too but I'm not sure where that's used. Anyways, it will be good to keep it current.

Reply:
Sounds good!